README.md

```diff
@@ -12,40 +12,43 @@
 
 A free-to-win rhythm game. Rhythm is just a *click* away!
 
-The future of [osu!](https://osu.ppy.sh) and the beginning of an open era! Currently known by and released under the release codename "*lazer*". As in sharper than cutting-edge.
+This is the future – and final – iteration of the [osu!](https://osu.ppy.sh) game client and marks the beginning of an open era! Currently known by and released under the release codename "*lazer*". As in sharper than cutting-edge.
```

Comment:
```suggestion
This is the future – and final – iteration of the [osu!](https://osu.ppy.sh) game client which marks the beginning of an open era! Currently known by and released under the release codename "*lazer*". As in sharper than cutting-edge.
```

sentence kinda doesn't flow otherwise (*This is the (...) final iteration (...) and marks the beginning of an open era*)

Reply:
Yeah I wasn't sure about this (two usages of `and` in a sentence). I'm not sure "which" makes much more sense, but it at least fixes this issue so let's go with it.